core/unix/signal.c

```diff
@@ -5784,7 +5790,7 @@ main_signal_handler_C(byte *xsp)
         /* For can_always_delay[sig] we could just return and drop it, but we
```

Comment:
Please update the comment at line 3529 which currently says:

```
        /* Handle DR's frame already being on the app stack.  For native delivery we
         * could try to re-use this frame, but that doesn't work with plain vs rt.
         * Instead we move below and live with the downsides of a potential stack
         * overflow.
         */
```

Reply:
Done